gtsfm/bundle/bundle_adjustment.py

```diff
@@ -56,6 +57,8 @@
 CAM_CAL3BUNDLER_PRIOR_NOISE_SIGMA = 1e-5  # essentially fixed
 CAM_CAL3FISHEYE_PRIOR_NOISE_SIGMA = 1e-5  # essentially fixed
 MEASUREMENT_NOISE_SIGMA = 1.0  # in pixels
+HARD_POSE_PRIOR_SIGMA = 1e-3  # 1e-5 did not work as well
+SOFT_POSE_PRIOR_SIGMA = 3e-2  # 1e-5 did not work as well
```

Comment:
can these sigmas instead be defined wherever we create the pose prior? 
We could create an artificial covariance matrix using these instead of hardcoding here.

Reply:
Is this not almost automatic with Hydra? It would be nice to have an explanation of how Hydra works in GTSFM for collaborators.

Reply:
Yes I can define them where I am creating the `PosePrior` object. I thought to just hardcode it as its a temporary solution.